lib/ruby_lsp/target_context.rb

```diff
@@ -0,0 +1,21 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  class TargetContext
+    extend T::Sig
+
+    sig { returns(T.nilable(Prism::Node)) }
+    attr_reader :closest, :parent
```

Comment:
nit: I prefer adding `_node` to them so it's immediately clear about what they returns, especially for addons that don't use Sorbet.

Reply:
Also, should we simply call `closest_node` `target_node` instead 🤔